Framework/Core/src/DDSConfigHelpers.cxx

```diff
@@ -117,6 +116,8 @@ void dumpDeviceSpec2DDS(std::ostream& out,
     }
   }
 
+  float timeout = 0.0;
```

Comment:
@davidrohr this starts from 0, in the chat you had 0.2, was that intended?

Reply:
no, from 0 is fine of course :)